apps/src/dance/ai/AiBlockPreview.tsx

```diff
@@ -0,0 +1,42 @@
+import {BlockSvg, Workspace, WorkspaceSvg} from 'blockly';
+import React, {useEffect, useRef} from 'react';
+import moduleStyles from './ai-block-preview.module.scss';
+
+interface AiPreviewProps {
```

Comment:
Nit: rename props here and in `AiVisualizationPreview` to distinguish?

Reply:
Done!